### ISSUE TITLE
Spec: Bring back the sidebar

### ISSUE THREAD

**Issue by Lamparter** [0x5bfa/FluentHub#507]
### 📄 Description

I really want to bring back the original sidebar from earlier versions that was inspired by the Microsoft Store as navigation is limited and this tool was incredibly useful to navigate between main pages.

### 🔙 Backward Compatibility

No changes to internal API contract/data storage/installation structure and therefore no issues to backwards compatibility.

### ✅ Tasks

- [ ] Seek internal approval
- [ ] Decide which design to use
- [ ] Implement design
- [ ] Add animations(?)

### 🤔 Difficulty

Low

**Comment by Lamparter** [comment]
> Nah...

😭 I really liked actually..

**Comment by Lamparter** [comment]
But @0x5bfa there needs to be a better way to navigate

**Comment by Lamparter** [comment]
@0x5bfa did you have a mock-up of what you wanted the hamburger menu to look like? Is it just like GitHub's one?

**Comment by 0x5bfa** [comment]
> did you have a mock-up of what you wanted the hamburger menu to look like? Is it just like GitHub's one?

Yeah I'd like to somewhat duplicate what we have in the dashboard of FluentHub.

**Comment by Lamparter** [comment]
Maybe we could implement a home button _here_ like in Files?
![image](https://github.com/user-attachments/assets/8662e30a-bea4-4c20-a7b7-5b106989fce9)

**Comment by Lamparter** [comment]
![image](https://github.com/user-attachments/assets/cb7e45d9-152f-4a6b-ad7a-0f66deeee94c)
Going well!

**Comment by 0x5bfa** [comment]
Why there? We should use what we have had. Also that hamburger menu should be number the current tab.
